MAINTAINERS.md

```diff
@@ -27,3 +27,57 @@ They will also be moved to the "former maintainers" section here.
 
 Former maintainers who want their access rights restored should open
 an issue or mail one of the active maintainers.
+
+## Maintainer guidelines
+
+The process for preparing and submitting changes as outlined
+in the [CONTRIBUTING document](./CONTRIBUTING.md) also applies to
+maintainers.  This ensures high quality contributions and keeps
+everybody on the same page.  Avoid direct pushes to the repository.
```

Comment:
We have a branch protection rule against pushes to main, so "Avoid" is a bit too weak.

```suggestion
everybody on the same page.  Do not make direct pushes to the repository.
```